File: WebAPI3/WebAPI3/Models/ActivityType.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;
using System.Text.Json.Serialization;

namespace WebAPI3.Models
{
    public class ActivityType
    {
        [Key]
        public int ActivityTypeId { get; set; }

        [Required]
        public string ActivityTypeName { get; set; }

        [JsonIgnore]
        public virtual ICollection<Activity> Activity { get; set; }
        [JsonIgnore]
        public virtual ICollection<UserActivityType> UserActivityType { get; set; }

    }
}

File: TimeAngul/src/app/shared/models/activity-type.ts
import { Activity } from "./activity.model";
import { UserActivityType } from './user-activity-type';

export interface ActivityType {
  ActivityTypeId: number;
  ActivityTypeName: string;
  Activity?: Activity[];
  UserActivityType: UserActivityType;
}

export interface ActivityTypeForm {
  ActivityTypeId?: number;
  ActivityTypeName: string;
  TimeFrom: number;
  TimeTo: number;
}
